docs/supercharge-your-dev-environment.md

```diff
@@ -0,0 +1,63 @@
+# Supercharge your dev environment
+
+Use this documentation to help supercharge your dev environment. We recommend using Rider or Visual Studio with ReSharper.
+
+- [Supercharge your dev environment](#supercharge-your-dev-environment)
+    - [Set up continuous testing](#set-up-continuous-testing)
+    - [Analyse test coverage](#analyse-test-coverage)
+    - [Configure linting and code cleanup](#configure-linting-and-code-cleanup)
+
+### Set up continuous testing
```

Comment:
Please can we review all the heading levels on all the new files (as we've essentially promoted all the text paragraphs) - for example, this heading should now be `##`